tests/expressions_case/tests.py

```diff
@@ -1243,3 +1243,31 @@ def test_conditional_aggregation_example(self):
             ),
             {'regular': 2, 'gold': 1, 'platinum': 3}
         )
+
+    def test_annotation_with_values_and_when(self):
```

Comment:
Ya ok,so then I'll remove the extra test.As far as the simplicity of tests us concerned,I feel the tests are included and the ones you included in the diff are pretty much the same.Do you want me to still remove my tests and use your tests?And about the existing tests in the docs,do you want me to remove them or add those to the expressions_case/tests.py ?